projects/igniteui-angular/src/lib/combo/combo.common.ts

```diff
@@ -27,6 +53,1244 @@ export interface IgxComboBase {
+     * ```
+     *
+     * ```html
+     * <!--set-->
+     * <igx-combo [allowCustomValues]='true'></igx-combo>
+     * ```
+     */
+    @Input()
+    public allowCustomValues = false;
+
+    /**
+     * Configures the drop down list height
+     *
+     * ```typescript
+     * // get
+     * let myComboItemsMaxHeight = this.combo.itemsMaxHeight;
+     * ```
+     *
+     * ```html
+     * <!--set-->
+     * <igx-combo [itemsMaxHeight]='320'></igx-combo>
+     * ```
+     */
+    @Input()
+    public get itemsMaxHeight(): number {
+        if (this._itemsMaxHeight === null || this._itemsMaxHeight === undefined) {
+            return this.itemHeight * itemsInContainer;
+        }
+        return this._itemsMaxHeight;
+    }
+
+    public set itemsMaxHeight(val: number) {
+        this._itemsMaxHeight = val;
+    }
+
+    /**
+     * Configures the drop down list item height
+     *
+     * ```typescript
+     * // get
+     * let myComboItemHeight = this.combo.itemHeight;
+     * ```
+     *
+     * ```html
+     * <!--set-->
+     * <igx-combo [itemHeight]='32'></igx-combo>
+     * ```
+     */
+    @Input()
+    public get itemHeight(): number {
+        if (this._itemHeight === null || this._itemHeight === undefined) {
+            return ItemHeights[this.displayDensity];
+        }
+        return this._itemHeight;
+    }
+
+    public set itemHeight(val: number) {
+        this._itemHeight = val;
+    }
+
+    /**
+     * Configures the drop down list width
+     *
+     * ```typescript
+     * // get
+     * let myComboItemsWidth = this.combo.itemsWidth;
+     * ```
+     *
+     * ```html
+     * <!--set-->
+     * <igx-combo [itemsWidth] = '"180px"'></igx-combo>
+     * ```
+     */
+    @Input()
+    public itemsWidth: string;
+
+    /**
+     * Defines the placeholder value for the combo value field
+     *
+     * ```typescript
+     * // get
+     * let myComboPlaceholder = this.combo.placeholder;
+     * ```
+     *
+     * ```html
+     * <!--set-->
+     * <igx-combo [placeholder]='newPlaceHolder'></igx-combo>
+     * ```
+     */
+    @Input()
+    public placeholder;
+
+    /**
+     * Defines the placeholder value for the combo dropdown search field
+     *
+     * ```typescript
+     * // get
+     * let myComboSearchPlaceholder = this.combo.searchPlaceholder;
+     * ```
+     *
+     * ```html
+     * <!--set-->
+     * <igx-combo [searchPlaceholder]='newPlaceHolder'></igx-combo>
+     * ```
+     */
+    @Input()
+    public searchPlaceholder = 'Enter a Search Term';
+
+    /**
+     * Defines whether the caseSensitive icon should be shown in the search input
+     *
+     * ```typescript
+     * // get
+     * let myComboShowSearchCaseIcon = this.combo.showSearchCaseIcon;
+     * ```
+     *
+     * ```html
+     * <!--set-->
+     * <igx-combo [showSearchCaseIcon]='true'></igx-combo>
+     * ```
+     */
+
+    @Input()
+    public showSearchCaseIcon = false;
```

Comment:
`showSearchCaseIcon` does nothing in simple combo. We should either remove it from common or make it work.